data/language/en-GB.yml

```diff
@@ -2410,3 +2410,6 @@ strings:
   2355: "{COLOUR WINDOW_2}Riverbank width:"
   2356: "{COLOUR WINDOW_2}Meander rate:"
   2357: "OpenLoco"
+  2358: "Toggle paint tool"
+  2359: "{SMALLFONT}{COLOUR BLACK}Repaint {POP16}{STRINGID} vehicles{NEWLINE}shift+click to paint all{NEWLINE}ctrl+click to paint sub-vehicles"
```

Comment:
The only issue with locking painting components behind the modifier is that mobile players (not that the game can be played on mobile yet) will not be able to do it.

Reply:
With the addition of a dropdown, this is no longer an issue, right?